docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerCloud.java

```diff
@@ -269,10 +265,11 @@ public Node call() throws Exception {
                                     //
                                     // deferring the completion of provisioning until the launch
```

Comment:
This also not my code, but without it i had spawned many containers. 
What will be the right way of provisioning/calling some methods for this block of code?
As i see Computer object created after slave added and created by update() calls after adding slave.